roles/pulp_health_check/README.md

```diff
@@ -4,6 +4,12 @@ pulp_health_check
 Verify if Pulp Services are up and listening.
 Note: this role is meant to be run on the host that [pulp_api](https://pulp-installer.readthedocs.io/en/latest/roles/pulp_api/) is run against
 
+Role Variables
+--------------
+
+* `pulp_default_host`: The host to be used as default when `pulp_api_bind` is an unix socket.
+  Defaults to `localhost`.
+
```

Comment:
It seems like the only reason we are setting it is because of some limitation uri module, correct?. The usual unix socket syntax looks like this:

http:/pulp/api/v3/status/

https://superuser.com/questions/834307/can-curl-send-requests-to-sockets

If there is such a limitation, is there any reason why a user would want to set this?

If not, let's make this a private variable or hardcode it. To make it private, prefix it with __ , and do not list in the README.md.

Reply:
I'm not sure about limitation, I've followed an uri example, I will test it

Reply:
yes, this is a `uri` limitation, as I could not think why a user would set this, I made it a private variable as you suggested